### DISCUSSION TITLE
lilygo t-embed

### DISCUSSION THREAD

**Discussion by LuxStatera** [landonr/homeThing#36] [Ideas]
https://www.lilygo.cc/products/t-embed

Would this device work with this project?

**Comment by landonr** [comment]
it uses the tdisplay s3 so no. The screen isn't supported by esphome. I have a tdisplay s3 so we either need to make a display driver for it, or use tft_espi

**Reply by Not5err** [reply]
Is this still the case? Seems like the t-embed is a nearly turnkey hardware solution for this project.

**Reply by landonr** [reply]
It should work using this config https://github.com/landonr/homeThing/pull/39

let me know how it works, I can update the docs. I ordered one and it was stolen from the mail
